File: web/GotoDN/src/GotoDN.Web/wwwroot/src/models/CategoryLanguageModel.ts
import {CategoryModel} from "./CategoryModel";
import {ImageModel} from "./ImageModel";
import {LanguageEnums} from "../commons/constant";
export interface CategoryLanguageModel {
  Id: number,
  CategoryId?: number,
  Title?: string,
  Icon?: string,
  ImageId?: number,
  CreatedDate?: Date,
  UpdatedDate?: Date,
  Language?: LanguageEnums,
  Category?: CategoryModel,
  Image?: ImageModel
}

File: web/GotoDN/src/GotoDN.Web/Models/CategoryLanguageModel.cs
using GotoDN.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GotoDN.Web.Models
{
    public class CategoryLanguageModel
    {
        public int Id { get; set; }
        public int? CategoryId { get; set; }
        public string Title { get; set; }
        public string Icon { get; set; }
        public int? ImageId { get; set; }
        public LanguageEnums? Language{ get; set; }

        public DateTime? CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
        
        public CategoryModel Category { get; set; }
        public ImageModel Image { get; set; }
    }
}
